docs/scripts_md/concat_mri.md

```diff
@@ -0,0 +1,62 @@
+# NAME
+concat\_mri.pl - aggregates minc files that have a common set of properties.
+
+# SYNOPSIS
+concat\_mri.pl \[options\] \[minc\_files\]
+
+- **-clean** : remove the original files when they are concatenated
+- **-clobber** : overwrite existing output file(s)
+- **-debug** : print commands but do not execute them
+- **-verbose** : print additional information while running
+- **-compress** : compress resulting output files with `gzip`
+- **-stdin** : do not read the names of the minc files on the commmand line but read them
+from STDIN instead
+- **-postfix \[postfix\]** : Create output file names using the first file name, followed by 
```

Comment:
to remain consistant with other description, "C" in "create" should be lower case. (very picky, I know...)

Reply:
Done